internal/driver/tempfile.go

```diff
@@ -21,16 +21,25 @@ import (
 	"sync"
 )
 
-// newTempFile returns an unused filename for output files.
-func newTempFile(dir, prefix, suffix string) (*os.File, error) {
+// newTempFilePath returns an unused path for output files.
+func newTempFilePath(dir, prefix, suffix string) (string, error) {
 	for index := 1; index < 10000; index++ {
```

Comment:
Save some duplication by calling newTempFile and run os.File.Name to get the filename (don't forget to close the file).